docker-compose.yml

```diff
@@ -1,17 +1,38 @@
-version: '3.1'
+version: '3.4'
 
 services:
+    frontend:
+        build:
+            context: .
+            dockerfile: ./Dockerfile.frontend.dev
+        ports:
+            - 5001:5001
+    chatserver:
+        build:
+            context: .
+            dockerfile: ./Dockerfile.dev
+        ports:
+            - 8080:8080
+        depends_on:
+            - postgrest
```

Comment:
@elimisteve I sorta skimmed your thread from last night in the email, re: networking. I don't think we need to use networking mode = "host" to expose things. The default bridge network should be sufficient.

I exposed port 8080 in the golang container and that allowed me to load via `127.0.0.1:8080`.

Reply:
You're right that it shouldn't be needed. I couldn't get port 8080 to appear open from my host without it, but I'm glad it worked for you and hopefully it works for me after I reboot so that my user is thoroughly in the `docker` group, and/or [toggle my VPN on or off](https://stackoverflow.com/a/57648759/197160), or something like that :sweat_smile: lol. #software

Reply:
To be clear: let's do what you said :+1: ; I'll figure out the networking shit on my end, and if I can't then I'll just run it on the host, which I've already got set up.